src/style.css

```diff
@@ -1 +1,77 @@
+/* stylelint-disable string-quotes */
+/* stylelint-disable color-hex-case */
+/* stylelint-disable no-missing-end-of-source-newline */
 /* Styles go here */
+
+body {
+  font-family: 'Roboto', Arial, Helvetica, sans-serif;
+  font-weight: 500;
+  padding: 0;
+  margin: 0;
+}
+
+.header {
+  padding: 0 50px;
+  max-width: 1200px;
+  display: flex;
+  margin: 0 auto;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo_link {
+  width: 40px;
+  height: 40px;
+}
+
+.moyo_nav__list {
+  margin: 0 auto;
+  padding: 0;
+  display: flex;
+  list-style: none;
+  align-items: center;
+}
+
+.moyo_nav__item {
+  margin-right: 20px;
+}
+
+.moyo_nav__item:last-child {
+  margin-right: 0;
+}
+
+.moyo_nav__link {
+  font-size: 12px;
+  position: relative;
+  height: 60px;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #000;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.moyo_nav__link:hover {
+  color: #00ACDC;
+}
+
+.is-active {
+  color: #00ACDC;
+}
+
+.is-active::after {
+  color: #00ACDC;
```

Comment:
add variables for colors you use, especially for values you using multiple times